### PR TITLE
Port `zerion.trades` to Dune v2

### PR DESCRIPTION
**Pull request by hildobby**
Brief comments on the purpose of your changes:

**For Dune Engine V2**

I've checked that:

### General checks:
* [ ] I tested the query on dune.com after compiling the model with dbt compile (compiled queries are written to the target directory)
* [ ] I used "refs" to reference other models in this repo and "sources" to reference raw or decoded tables 
* [ ] if adding a new model, I added a test
* [ ] the filename is unique and ends with .sql
* [ ] each sql file is a select statement and has only one view, table or function defined  
* [ ] column names are `lowercase_snake_cased`
* [ ] if adding a new model, I edited the dbt project YAML file with new directory path for both models and seeds (if applicable)
* [ ] if wanting to expose a model in the UI (Dune data explorer), I added a post-hook in the JINJA config to add metadata (blockchains, sector/project, name and contributor Dune usernames)

### Pricing checks:
* [ ] `coin_id` represents the ID of the coin on coinpaprika.com
* [ ] all the coins are active on coinpaprika.com (please remove inactive ones)

### Join logic:
* [ ] if joining to base table (i.e. ethereum transactions or traces), I looked to make it an inner join if possible

### Incremental logic:
* [ ] I used is_incremental & not is_incremental jinja block filters on both base tables and decoded tables
  * [ ] where block_time >= date_trunc("day", now() - interval '1 week')
* [ ] if joining to base table (i.e. ethereum transactions or traces), I applied join condition where block_time >= date_trunc("day", now() - interval '1 week')
* [ ] if joining to prices view, I applied join condition where minute >= date_trunc("day", now() - interval '1 week')

### CODE REVIEW

**Comment by jeff-dude** [comment]
closing due to inactivity, can be reopened as needed in future

**Review by Hosuke — APPROVED** [review]
LGTM.✅
Thank you @hildobby 

Currently zerion.trades is not added into dex.trades.

[Sample zerion.trades](https://dune.com/queries/2620772?d=1)

File: models/zerion/arbitrum/zerion_arbitrum_trades.sql [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+    , tok_bought.decimals AS tok_bought_decimals
+    , pt.from AS tx_from
+    , pt.to AS tx_to
+    , swap.evt_tx_hash AS tx_hash
+    , swap.contract_address
+    , swap.evt_index
+    , swap.marketplaceFeeAmount AS marketplace_fee_amount_raw
+    , swap.protocolFeeAmount AS zerion_fee_amount_raw
+    FROM {{ source('zerion_arbitrum', 'Router_evt_Executed') }} swap
+    INNER JOIN {{ source('arbitrum','transactions') }} pt ON pt.block_number=swap.evt_block_number
```
Comment (jeff-dude) [inline models/zerion/arbitrum/zerion_arbitrum_trades.sql:45]: can you add the `is incremental` and `is not incremental` filters in the join conditions to base tables too, similar to prices below?

**Review by jeff-dude — CHANGES_REQUESTED** [review]
in addition to below, as hosuke mentioned, this isn't integrated into `dex.trades` through this PR. is that the intention in the future? if so, we'd want to match the columns. it's fine to have a few extra columns at the base project models, but we'll need to include the columns for dex for sure

**Comment by hildobby** [comment]
> in addition to below, as hosuke mentioned, this isn't integrated into `dex.trades` through this PR. is that the intention in the future? if so, we'd want to match the columns. it's fine to have a few extra columns at the base project models, but we'll need to include the columns for dex for sure

No, this is meant to be an independent table, not feed into dex.trades. Zerion aggregates the trades on their frontend but uses other DEXs under the hood so it could be into aggregators but it's also a meta aggregator using 1inch and matcha sometimes so I think it's easier to leave it independent for now

File: models/zerion/arbitrum/zerion_arbitrum_trades.sql (hunk 1 above)
Comment (hildobby) [reply models/zerion/arbitrum/zerion_arbitrum_trades.sql:45]: Done!

**Review by jeff-dude — APPROVED** [review]
sounds good

**Comment by hildobby** [comment]
Talked to the Zerion team who just submitted their Fantom contract for decoding on fantomscan, I just submitted it on dune and will add fantom to this PR once it's been decoded by @antonio-mendes (it's the exact same code as for the other chains). I will mark this PR as do-not-merge in the meantime

**Comment by antonio-mendes** [comment]
@hildobby https://dune.com/dune/is-my-contract-decoded-yet-v2?contract_ta8992=0xd7f1dd5d49206349cae8b585fcb0ce3d96f1696f contract is live on Fantom 👍

**Comment by hildobby** [comment]
> @hildobby https://dune.com/dune/is-my-contract-decoded-yet-v2?contract_ta8992=0xd7f1dd5d49206349cae8b585fcb0ce3d96f1696f contract is live on Fantom 👍

Awesome thanks @antonio-mendes! I've added Fantom to the PR :)

**Pull request merged**